pkgs/by-name/ma/mani/package.nix

```diff
@@ -1,9 +1,9 @@
 {
+  lib,
+  git,
   buildGoModule,
   fetchFromGitHub,
-  lib,
   installShellFiles,
-  git,
   makeWrapper,
 }:
 
```

Comment:
If you want you can get rid of `rec` attribute, `buildGoModule` supports `finalAttrs` pattern now.

Reply:
done, thanks